public/data/2270657-3743/info.json

```diff
@@ -0,0 +1,44 @@
+{
+  "user": {
+    "username": "OscarDavidCLOP",
+    "profileImage": "https://lh3.googleusercontent.com/a/ACg8ocKNqjYnOYxFBQYru-92R3_brp08jq4rPjByksGyP8neJuWEaQ=s288-c-no",
+    "socialLink": "@david37_2"
+  },
+  "topics": [
+    {
+      "topic": "grammar",
+      "questions": [
+        {
+          "question": "Which sentence is in the past tense?",
+          "options": [
+            "I am reading a book.",
+            "I read a book yesterday.",
+            "I will read a book tomorrow.",
+            "I have been reading a book."
+          ],
+          "answer": "I read a book yesterday."
+        },
+        {
+          "question": "Choose the correct form of the verb: 'She ___ to the store every day.'",
+          "options": ["go", "goes", "gone", "going"],
+          "answer": "goes"
+        }
+      ]
+    },
+    {
+      "topic": "vocabulary",
+      "questions": [
+        {
+          "question": "What is the antonym of 'hot'?",
+          "options": ["warm", "cold", "humid", "mild"],
+          "answer": "cold"
+        },
+        {
+          "question": "Translate 'dog' into French.",
```

Comment:
Come on! Questions should be in an English context only, not in French. :(

Reply:
@oscar-230 Could you please change this question ? should be in an English context only